app/Spiders/FacultySpider.php

```diff
@@ -58,17 +62,19 @@ public function parseFields(Response $response): Generator
             $text = $crawler->text();
 
             return [
-                "name" => substr($text, 0, strpos($text, "-")),
+                "name" => substr($text, 0, strpos($text, "-") - 2),
+                "year" => substr($text, strpos($text, "-") - 1, 1),
                 "slug" => substr($text, strpos($text, "(") + 1, -1),
-                "isFullTime" => substr($text, strpos($text, "(") + 1, 1) === "s",
+                "type" => substr($text, strpos($text, "(") + 1, 1),
             ];
         });
 
         foreach ($fields as $field) {
             $fieldItem = new FieldItem(
                 $field["name"],
+                intval($field["year"]),
                 $field["slug"],
-                $field["isFullTime"],
+                $field["type"] === "s",
```

Comment:
why s ?

Reply:
S = full-time studies
N = part-time studies

Reply:
Maybe is a create enum for this value in next mr ?Becouse s for me is a magic string and you can make a type in the database from this enum and save it like this